src/transformers/modeling_flax_utils.py

```diff
@@ -333,6 +423,38 @@ def to_fp16(self, params: Union[Dict, FrozenDict], mask: Any = None):
         ```"""
         return self._cast_floating_to(params, jnp.float16, mask)
 
+    @classmethod
+    def load_flax_sharded_weights(cls, shard_files):
+        """
+        Args:
+        This is the same as
+        [`flax.serialization.from_bytes`](https:
+            //flax.readthedocs.io/en/latest/_modules/flax/serialization.html#from_bytes)
+        but for a sharded checkpoint. This load is performed efficiently:
+            each checkpoint shard is loaded one by one in
+        RAM and deleted after being loaded in the model.
+            shard_files (`List[str]`:
+                The list of shard files to load.
```

Comment:
(nit) the doc formatting looks a bit weird here.

Reply:
Reformatted it